transcend/resource_data_silo.go

```diff
@@ -365,6 +365,11 @@ func resourceDataSilo() *schema.Resource {
 			// 	},
 			// 	Description: "The names of the teams that should be responsible for this data silo",
 			// },
+			"sombra_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
My only question here is what we would do if someone theoretically wanted to change the sombra_id for a given data silo. Do we support that? Or would we want to replace the entire data silo and assign the new data silo to the updated sombra id?

Reply:
We do support it! Someone can just include a new sombra_id and then the update mutation will just update the sombra_id in postgres.